### PR TITLE
logstash_plugin - invoke run_command passing list

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Passing command as list instead of as string.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/modules/monitoring/logstash_plugin.py

### CODE REVIEW

File: changelogs/fragments/3805-iso_extract-run_command-list.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+    - iso_extract - calling ``run_command`` with arguments as ``list`` instead of ``str`` (https://github.com/ansible-collections/community.general/pull/3805).
```
Comment (felixfontein) [inline changelogs/fragments/3805-iso_extract-run_command-list.yaml:2]: This shouldn't part of this PR.

Comment (russoz) [reply changelogs/fragments/3805-iso_extract-run_command-list.yaml:2]: oh shoot

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russo thanks!
